priv/features/contexts/white_bread_context.exs

```diff
@@ -164,11 +182,36 @@ defmodule WhiteBreadContext do
     end
   )
 
-  def assert_equal(left, right) do
+  # IFE
+  when_(~r/Alice starts an in flight exit$/, fn %{
+                                                  alice_account: alice_account,
+                                                  alice_pkey: alice_pkey,
+                                                  bob_account: bob_account
+                                                } = state,
+                                                _ ->
+    _ife = InFlightExitClient.start_in_flight_exit(alice_account, alice_pkey, bob_account)
+
+    {:ok, state}
+  end)
+
+  # IFE
+  then_(~r/Alice should have "(?<amount>[^"]+)" ETH after finality margin$/, fn %{alice_account: alice_account} = state,
+                                                                                %{amount: amount} ->
+    # Process.sleep(@finality_margin_by_blocks * 500 + 120_000)
```

Comment:
Shouldn't that line be deleted?

Reply:
indeed! it slipped! I'll clean this up in the next PR.